client/src/TitleScreen/TitleScreen.css

```
@@ -386,3 +386,31 @@
 	background-color: #121212;
 	transition: all 100ms ease-in-out;
 }
+.Credits-Heading {
+    font-family: 'VT323', monospace;
+    font-size: 50px;
+    color: white;
+    
+    text-align: center;
+}
+body {
+    background-color: black;
+    width: 50vw;
+}
+.credit-title {
```

Append `.Credits-Heading` in front of all the child elements of `Credits-Heading` element.
```suggestion
.Credits-Heading .credit-title {
```